lexicons/com/atproto/sync/subscribeRepos.json

```diff
@@ -206,6 +206,10 @@
         "cid": {
           "type": "cid-link",
           "description": "For creates and updates, the new record CID. For deletions, null."
+        },
+        "prev": {
+          "type": "cid-link",
+          "description": "EXPERIMENTAL. For deletes and updates, the CID  of the previous record. For creates, undefined."
```

Comment:
Should we use a `nullable` to be able to determine if the implementation supports this or not ?

Reply:
I had the same thought about detecting support.  Daniel pointed out that we don't need to worry about that yet for the experimental phase.  The tricky part here will be that we want to avoid optional, nullable fields.

Reply:
just to note, `nullable` and `optional` combined is a no-go (for golang), and that is especially relevant for these wire protocol details.